api/src/service/domain/workflow/workflowitem_update.spec.ts

```diff
@@ -6,7 +6,7 @@ import { BusinessEvent } from "../business_event";
 import { NotAuthorized } from "../errors/not_authorized";
 import { NotFound } from "../errors/not_found";
 import { ServiceUser } from "../organization/service_user";
-import { DocumentReference, hashDocument } from "../document/document";
+import { DocumentReference, UploadedDocument, hashDocument } from "../document/document";
```

Comment:
also here both `UploadedDocument, hashDocument` are not used

Reply:
Fixed